src/modules/GroupChannelList/components/GroupChannelListItem/utils.ts

```diff
@@ -10,7 +10,7 @@ export const getChannelTitle = (channel?: GroupChannel, currentUserId?: string,
   if (!channel?.name && !channel?.members) {
     return stringSet.NO_TITLE;
   }
-  if (channel?.name && channel.name !== 'Group Channel') {
+  if (channel?.name && channel.name !== 'Group Channel' && channel.name !== 'AI Chatbot Widget Channel') {
```

Comment:
Here is the primary fix. The rest of the PR is getting all consumers to use a single getChannelTitle helper function